src/gui/tray/Window.qml

```diff
@@ -53,6 +62,15 @@ Window {
             folderStateIndicator.source = UserModel.isUserConnected(UserModel.currentUserId)
                     ? Style.stateOnlineImageSource
                     : Style.stateOfflineImageSource
+            
+            currentAccountStatusIndicator.source = ""

```

Comment:
@camilasan can this be set on the last line where you're checking for `currentUserStatus.visible`? Using ternary if condition if possible?